manifests/params.pp

```diff
@@ -48,7 +48,23 @@
 
   case $::osfamily {
     'Debian': {
-      $service_provider = 'upstart'
+
+      if $install_method == 'archive' {
+        $manage_unzip = true
+      }
+
+      case $::lsbdistcodename {
+        /(jessie|stretch|sid|xenial|yakketi|zesty)/: {
```

Comment:
Does this mean we can add support for those Ubuntu distros in `metadata.json` too? Or is there still some work to do for those?